src/components/molecules/CameraMicrophoneControls/CameraMicrophoneControls.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useCallback } from "react";
+import classNames from "classnames";
+import Video, {
+  AudioTrackPublication,
+  VideoTrackPublication,
+} from "twilio-video";
+
+import { ContainerClassName } from "types/utility";
+
+import { VideoOverlayButton } from "components/atoms/VideoOverlayButton";
+
+import "components/molecules/CameraMicrophoneControls/CameraMicrophoneControls.scss";
+
+export interface CameraMicrophoneControlsProps extends ContainerClassName {
+  participant: Video.Participant;
+  defaultMute: boolean;
+}
+
+export const CameraMicrophoneControls: React.FC<CameraMicrophoneControlsProps> = ({
+  participant,
+  defaultMute,
+  containerClassName,
+}) => {
+  const getChangeStateHandler = useCallback(
+    (variant: "microphone" | "camera"): ((enabled: boolean) => void) => {
+      const tracks =
+        variant === "microphone"
+          ? participant.audioTracks
+          : participant.videoTracks;
+
+      return (enable: boolean) => {
+        tracks.forEach(
+          (track: AudioTrackPublication | VideoTrackPublication) => {
+            const innerTrack = track.track;
+            if (innerTrack) {
+              if (enable && "enable" in innerTrack) innerTrack.enable();
+              else if ("disable" in innerTrack) innerTrack.disable();
```

Comment:
`in` traverses prototype.

Isn't `innerTrack.enable` good enough? 
```ts
const keys = Object.keys(innerTrack)
if(enable && keys.includes("enable")) //...
```

Reply:
With `keys` TS complaints on `innerTrack.enable()` as with `in` it can statically check `enable` invocation.

Reply:
Yep, I've checked... TS quirks... https://github.com/sparkletown/sparkle/pull/2231#discussion_r701616106